src/Transports.AspNetCore/GraphQLHttpMiddleware.cs

```diff
@@ -67,11 +62,15 @@ public async Task InvokeAsync(HttpContext context)
                 switch (mediaTypeHeader.MediaType)
                 {
                     case JsonContentType:
-                        if (!Deserialize(httpRequest.Body, out gqlRequest, out gqlBatchRequest))
+                        var deserializationResult = await _deserializer.DeserializeFromJsonBodyAsync(httpRequest, GetCancellationToken(context))
```

Comment:
Please confirm this is correct.

Reply:
Move `var token = GetCancellationToken(context);` higher and pass it into `DeserializeFromJsonBodyAsync`

Reply:
Didn't do this initially in case there was work involved in generating that token, but traced through the code and it's just a plain property so all good. Done.